.vscode/settings.json

```diff
@@ -1,87 +1,93 @@
 {
-    "files.associations": {
-        "*.metaed": "csharp",
-        "*.hcl": "hcl"
+  "files.associations": {
+    "*.metaed": "csharp",
+    "*.hcl": "hcl"
+  },
+  "psi-header.config": {
+    "forceToTop": true,
+    "blankLinesAfter": 1,
+    "license": "Custom"
+  },
+  "psi-header.templates": [
+    {
+      "language": "*",
+      "template": [
+        "SPDX-License-Identifier: Apache-2.0",
+        "Licensed to the Ed-Fi Alliance under one or more agreements.",
+        "The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.",
+        "See the LICENSE and NOTICES files in the project root for more information."
+      ]
+    }
+  ],
+  "psi-header.lang-config": [
+    {
+      "_": "Default settings for other languages, such as HCL",
+      "language": "*",
+      "begin": "",
+      "end": "",
+      "prefix": "# "
```

Comment:
This is the real change. The file was also reformatted by VS Code based on the EditorConfig settings.